how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,113 @@
+
+# Kata Containers with NEMU
+
+* [How to get started](#how-to-get-started)
+    * [Pre-requisites](#pre-requisites)
+    * [NEMU](#nemu)
+        * [Download, build and install](#download-build-and-install)
+    * [Configure Kata Containers](#configure-kata-containers)
+
+Kata Containers relies by default on the QEMU hypervisor in order to spawn the virtual machines running containers. [NEMU](https://github.com/intel/nemu) is a fork of QEMU that:
+- Reduces the number of lines of code.
+- Removes all legacy devices.
+- Reduces the emulation as far as possible.
+
+## How to get started
+
+This section describes how to run Kata Containers with NEMU, first by explaining how to download, build and install it. Then this document walks through the steps needed to update your Kata Containers configuration in order to run with NEMU. 
+
+### Pre-requisites
+This document requires Kata Containers to be [installed](https://github.com/kata-containers/documentation/blob/master/install/README.md) on your system.
+
+Also, it's worth noting that NEMU only supports `x86_64` and `aarch64` architecture.
+
+### NEMU
+
+#### Download, build and install
+
+```bash
+$ git clone https://github.com/intel/nemu.git
+$ cd nemu
+$ git fetch origin
+$ git checkout origin/experiment/automatic-removal
+```
+__x86_64__
```

Comment:
Can you make this and the one below either sub-sections of the "NEMU" section or create a new "Install" section that they can hang off?